### DISCUSSION TITLE
b-Icon Accepts a single type input, but should allow for combining classes

### DISCUSSION THREAD

**Discussion by hkd987** [buefy/buefy#3996] [Ideas]
### Overview of the problem

**Buefy** version: 0.9.2
**Vuejs** version: 2.6.11
**OS/Browser**: MacOS -> Chrome 85

### Description

When using b-icon type accepts a single class input such as 

```
                  <b-icon
                    icon="edit"
                    type="is-warning"
                  ></b-icon>
```

One would expect to be able to add the is-light class to the type also however this input is not respected. 

```
                  <b-icon
                    icon="edit"
                    type="is-warning is-light"
                  ></b-icon>
```

**Comment by jtommy** [comment]
At the moment you might use `type="is-warning-light"`

**Comment by hkd987** [comment]
Since other components allow passing in multiple types, I would say this isn't an improvement but a bug fix. I would expect all type properties to function the same as a user.
